addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -461,10 +461,28 @@
 						<label>$INFO[ListItem.Label]</label>
 					</control>
 					<control type="image">
+						<visible>ListItem.IsPremiere</visible>
 						<left>6</left>
 						<top>35</top>
-						<width>20</width>
-						<height>20</height>
+						<width>16</width>
+						<height>16</height>
+						<aspectratio>keep</aspectratio>
+						<texture colordiffuse="button_focus">icons/pvr/premiere.png</texture>
+					</control>
+					<control type="image">
+						<visible>ListItem.IsPremiere</visible>
+						<left>25</left>
+						<top>35</top>
+						<width>16</width>
+						<height>16</height>
+						<texture>$VAR[PVRTimerIcon]</texture>
+					</control>
+					<control type="image">
+						<visible>!ListItem.IsPremiere</visible>
+						<left>6</left>
+						<top>35</top>
+						<width>16</width>
+						<height>16</height>
 						<texture>$VAR[PVRTimerIcon]</texture>
 					</control>
```

Comment:
you can use a single image control for the PVRTimerIcon:
```
<control type="image">
	<left>6</left>
	<top>35</top>
	<width>16</width>
	<height>16</height>
	<texture>$VAR[PVRTimerIcon]</texture>
	<animation effect="slide" start="0,0" end="19,0" condition="ListItem.IsPremiere">Conditional</animation>
</control>
```

Reply:
Nice. Will change that, ofc.

Reply:
Fixed.